src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -456,7 +456,7 @@ const Analyses = _.flow(
         [!_.isEmpty(analyses) && _.isEmpty(renderedAnalyses), () => {
           return div({ style: { fontStyle: 'italic' } }, ['No matching analyses'])
         }],
-        [Utils.DEFAULT, () => h(Fragment, [
+        [Utils.DEFAULT, () => div({ role: 'table' }, [
```

Comment:
Hmm, I'm not sure about putting a role of table on this, given that it contains a div with a role of list. What was the thought here?

Reply:
I believe what's happening here is that the list is acting the role of a table, so it needs the table role in order to have a the aria-sort appended to it(and work correctly)

Reply:
It does not behave well with a screen reader though because the things that are within the "table" do not behave as rows and columns. The screen reader announces that it is a table with 3 columns and one row, but you can't navigate through the columns at all. It doesn't makes sense to declare something as a list without also declaring roles of row and column.

Reply:
To do this properly, you will need to remove the role of list on the content, and add roles of cell and row. The groups page does a reasonable job of this, so I would recommend you take a look at it. https://bvdp-saturn-dev.appspot.com/#groups